.ci_support/linux_64_.yaml

```diff
@@ -1,8 +1,10 @@
 cdt_name:
-- cos7
+- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

Comment:
Why do you change this to 3.9? The repo still specifies 3.8 https://github.com/mosquito/aiormq/blob/master/pyproject.toml

Reply:
This was autogenerated in the re-rendering commit as per [CFEP-25](https://github.com/conda-forge/cfep/blob/main/cfep-25.md), see also PR #58 for details. Seems like 3.9 is the last official minimum python version. Given that this repo uses an automerge bot, I think it would be wise to adhere to CFEP-25 policy to prevent wrong `python_min` being set for the package. But for now I've added an explicit minimum version 3.8 to `recipe/meta.yaml` that should override this.